lib/Plerd/Post.pm

```diff
@@ -512,7 +525,9 @@ sub _process_source_file {
     if ( $attributes_need_to_be_written_out ) {
         my $new_content = '';
         for my $attribute_name ( @ordered_attribute_names ) {
-            $new_content .= "$attribute_name: $attributes{ $attribute_name }\n";
```

Comment:
This change seems unrelated to the tag-specific work?

Reply:
This change was needed to silence warnings in the test.  This wasn't a delete, but an existence/defined check (see below).   There is no harm input validation, is there?

Reply:
Okay, makes sense, thanks.

May as well just collapse the if-check to `defined $attributes{ $attribute_name }` though, right?